crates/biome_js_analyze/tests/specs/nursery/noEnum/invalid.ts.snap

```diff
@@ -23,7 +23,7 @@ invalid.ts:1:1 lint/nursery/noEnum ━━━━━━━━━━━━━━━
       │ ^
     4 │ 
   
-  i Prefer using JavaScript objects or TypeScript unions over enums.
+  i Prefer using JavaScript objects, TypeScript unions or const enums over plain enums.
```

Comment:
`const enum` have some important caveat (cf `noConstEnum`, docs). Moreover, we recommend `noConstEnum`.
Thus, I think it is not a good idea to suggest using them here.

```suggestion
  i Prefer using JavaScript objects or TypeScript unions over plain enums.
```